CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Changelog for NeoFS Node
 - `wallet-address` flag in `neofs-adm morph refill-gas` command (#1820)
 - Validate policy before container creation (#1704)
 - `--timeout` flag in `neofs-cli` subcommands (#1837)
+- Shards can be marked with `mode: disabled` to be ignored by node (#1840)
```

Comment:
I would prefer to leave more general desc like `[Added] - Flag to ignore shard in storage node config` and place details in `Updating...` section. But your variant is clear, too.

Reply:
Fixed